setup.py

```diff
@@ -12,7 +12,7 @@
 try:
     with open('requirements.txt') as requirements_file:
         DEPENDENCIES = requirements_file.readlines()
-except FileNotFoundError:
+except IOError:
```

Comment:
Understand `IOError` is available in Python 2.

But the Python 3 exception seems not to be caught from this:
```
Python 3.5.2 (default, Nov 23 2017, 16:37:01) 
>>> open('nothing')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
FileNotFoundError: [Errno 2] No such file or directory: 'nothing'
>>> FileNotFoundError.__bases__
(<class 'OSError'>,)
>>> OSError.__bases__                                                           
(<class 'Exception'>,)
```

Reply:
I check that too 👍  it's actually assigned in the `builtins.py` on `line 956` as:
`IOError = OSError`
To maintain compatibility with Python 2 `IOError` I believe.

If it was not for the duel support needed I would just leave it as `FileNotFoundError`.